packages/ui/src/layout/banners/Announcement.tsx

```diff
@@ -52,15 +53,20 @@ const Announcement = ({
   }
 
   function handleLink() {
-    router.push(announcement.link)
+    // redirect to home page if on docs subpage
+    if (isDocsSubPage) {
+      window.location.replace(`https://supabase.com${announcement.link}`)
```

Comment:
Let's call `assign` instead of `replace` so that the current page remains in the browser history. Can we also make this relative?
```suggestion
      window.location.assign(announcement.link)
```